.github/workflows/ci.yaml

```diff
@@ -119,6 +133,9 @@ jobs:
             contracts/precompiles/artifacts
             bootloader/build
 
+      - name: Starting test node
+        run: era_test_node run > /dev/null 2>&1 &
+
       - name: Run tests
         run: yarn test
```

Comment:
suggestion/tip: Always print out the console output of `era_test_node` logs just in case there's a failure, it will help with diagnosis

```yaml
- name: Print output logs of era_test_node
  if: always()
  run: cat era_test_node.log
```